devtools/server/actors/replay/redux-devtools/page.bundle.js

```diff
@@ -10360,6 +10362,9 @@ function __REDUX_DEVTOOLS_EXTENSION__(config) {
       reportId = (0,_app_stores_enhancerStore__WEBPACK_IMPORTED_MODULE_1__.getUrlParam)('remotedev_report');
       if (reportId) (0,_app_api_openWindow__WEBPACK_IMPORTED_MODULE_7__["default"])();
     }
+
+      console.log('Posting start message')
```

Comment:
:hand: This one I did leave in intentionally, but it will need to be removed as well.